src/controllers/update/overseas.entity.review.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData } from "../../utils/application.data";
+import { Session } from "@companieshouse/node-session-handler";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.OVERSEAS_ENTITY_REVIEW_PAGE}`);
+
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+    const backLinkUrl: string = config.OVERSEAS_ENTITY_REVIEW_PAGE; // to be changed
+
+    return res.render(config.OVERSEAS_ENTITY_REVIEW_PAGE, {
+      templateName: config.OVERSEAS_ENTITY_REVIEW_PAGE,
+      backLinkUrl,
+      appData
```

Comment:
pass in  OE_CONFIGS.OVERSEAS_ENTITY_UPDATE_DETAILS_URL via another variable so that the view can be parameterized by the journey kind.